firebase_admin/ml.py

```diff
@@ -52,6 +52,9 @@
 _TAG_PATTERN = re.compile(r'^[A-Za-z0-9_-]{1,60}$')
 _GCS_TFLITE_URI_PATTERN = re.compile(
     r'^gs://(?P<bucket_name>[a-z0-9_.-]{3,63})/(?P<blob_name>.+)$')
+_AUTO_ML_MODEL_PATTERN = re.compile(
+    r'^projects/(?P<project_id>\d+)/locations/(?P<location_id>[^/]+)/' +
```

Comment:
Project ID group is defined as `(?P<project_id>[a-z0-9-]{6,30})` in some of the other regex patterns in this file.

Reply:
Done.